MultiArrayLayout layout
int32[] WhiteRealDis
